receiver/datadogreceiver/translator.go

```diff
@@ -84,6 +84,7 @@ func toTraces(payload *pb.TracerPayload, req *http.Request) ptrace.Traces {
 			newSpan.SetParentSpanID(uInt64ToSpanID(span.ParentID))
 			newSpan.SetName(span.Name)
 			newSpan.Status().SetCode(ptrace.StatusCodeOk)
+			newSpan.Attributes().PutStr("span.Resource", span.Resource)
```

Comment:
I guess the name chosen here is probably the thing worth bikeshedding over? Could call it `dd.span.Resource` or something... 🤷

Reply:
I agree. People can use the transform processor if they want another name for this.